packs/equipment-effects/effect-boulderhead-bock.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Boulderhead Bock",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Boulderhead Bock].</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Boulderhead Bock].</p>\n<p>You gain a +1 item bonus to saving throws against effects that would make you stunned or stupefied.</p>\n<p>You take a -2 item penalty to Perception for 1 hour.</p>"
```

Comment:
Consider updating the predicate to `"inflicts:stunned"` and `"inflicts:stupefied"`.